manifest.go

```diff
@@ -0,0 +1,8 @@
+package f3
+
+import "github.com/filecoin-project/go-f3/gpbft"
+
+type Manifest struct {
+	NetworkName       gpbft.NetworkName
+	InitialPowerTable []gpbft.PowerEntry
```

Comment:
I recommend adding JSON struct tags to hardcode field names if the future production API will include this.

Reply:
This isn't even close to the final shape. Once we know what we want there, we can hardcode field names.